### ISSUE TITLE
Animated content cannot be stopped or paused [Android]

### ISSUE THREAD

**Issue by jonnyjames** [covid-alert-ny/covid-green-app#364]
"How it works" screen 1 of 4 contains animated content that doesn't have a control for users to either stop or pause it. This is an issue under WCAG Success Criterion **2.2.2 Pause, Stop, Hide (Level A)**: For moving, blinking, scrolling, or auto-updating information, all of the following are true:

**Moving, blinking, scrolling**
For any moving, blinking or scrolling information that (1) starts automatically, (2) lasts more than five seconds, and (3) is presented in parallel with other content, there is a mechanism for the user to pause, stop, or hide it unless the movement, blinking, or scrolling is part of an activity where it is essential; and

**Auto-updating**
For any auto-updating information that (1) starts automatically and (2) is presented in parallel with other content, there is a mechanism for the user to pause, stop, or hide it or to control the frequency of the update unless the auto-updating is part of an activity where it is essential.

![How it works - Animated - No pause](https://user-images.githubusercontent.com/7034772/92525211-6c068380-f1f1-11ea-962a-8eb184997915.png)

**Remediation Guidance**
Remove the animated content or include a control for users to pause/stop the animation.

**Comment by floridemai** [comment]
@helloantoine Can you jump in on this one pelase?

**Comment by helloantoine** [comment]
@fahd-arshad @ahurworth 
The animation lasts 8s. However, the character animation last 5s. and the following 3s. are only there to avoid looping too quickly. 

2 options:
A. I could kill the 'static' 3 second at the end, which means we'd fall under the 5s category.
B. We add a control on the video (not my favourite).

**Comment by ahurworth** [comment]
@fahd-arshad, I discussed this with @helloantoine.  We will need to kill the looping as well unfortunately.  But on balance, this decision feels better than adding a play/pause control.  IMHO, this would set the wrong expectation with the user that this is a video with more detailed explanations, not a basic animation.
The other alternative would be to make it a static image, which would be a shame.

**Comment by fahd-arshad** [comment]
Agreed. Let's go with A) get to 5 seconds, no looping. Can we at least restart the animation if the user re-enters the screen? We can't have it play just once period with no ability to review it.

**Comment by helloantoine** [comment]
Just seeing this now. Will do thanks!

**Comment by helloantoine** [comment]
PR accepted.
